at_hello_world/lib/service/client_sdk_service.dart

```diff
@@ -119,13 +115,13 @@ class ClientSdkService {
           .getAtKeys(regex: conf.MixedConstants.NAMESPACE, sharedBy: sharedBy);
 
   /// Fetches atsign from device keychain.
-  Future<String?> getAtSign() async => atClientServiceInstance!.getAtSign();
+  Future<String?> getAtSign() async => _keyChainManager.getAtSign();
 
   Future<void> deleteAtSignFromKeyChain() async {
     // List<String> atSignList = await getAtsignList();
-    String? atsign = atClientServiceInstance!.atClient!.currentAtSign;
+    String? atsign = atClientInstance.atClient.getCurrentAtSign();
 
-    await atClientServiceMap[atsign]!.deleteAtSignFromKeychain(atsign!);
+    await _keyChainManager.deleteAtSignFromKeychain(atsign!);
```

Comment:
getAtClientForAtsign()!.notify(..) is deprecated..use notificationService.subscribe.listen(..)